src/commissaire/containermgr/trivial.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2017  Red Hat, Inc
```

Comment:
If that isn't much extra work let's do that. If it is a pain then let me know and I won't block.